engines/wage/util.cpp

```diff
@@ -53,10 +53,10 @@ namespace Wage {
 Common::Rect *readRect(Common::SeekableReadStream *in) {
 	int x1, y1, x2, y2;
 
-	y1 = in->readSint16BE();
-	x1 = in->readSint16BE();
-	y2 = in->readSint16BE() + 4;
-	x2 = in->readSint16BE() + 4;
+	y1 = in->readSint16BE() - 2;
```

Comment:
Please explain in a comment why we're doing this

Reply:
Done